docs/topics/mta-web-assessment-module.adoc

```diff
@@ -0,0 +1,111 @@
+* Assessment module integrates tightly with the application inventory.
+* Assessment questionnaires can be imported and exported.
+* Enhanced questionnaire capabilities:
+** Conditional questions
+** Application auto-tagging according to answers
+** Automated answers from tags in application archetypes
+
+In the MTA user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved default and custom questionnaires.
+
+:_content-type: PROCEDURE
+[id="mta-assessment-start-questionnaire_{context}"]
+== Opening an assessment questionnaire
+
+The {ProductName} ({ProductShortName}) assesses applications that are to be migrated according to a set of questions relevant to the application, such as dependencies, *[more question types]*. {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.
+
+.To import an application
+
+// Get updated image for MTA
+image::mta-assessment-apps-01.png[MTA application inventory]
+
+. Open the {ProductName} user interface and click *Application Inventory* in the left menu bar. *Application assessment* appears in the main pane, listing all the available questionnaires. 
+. Click *Create new*. A *New application* dialog appears.
+
+// Get updated image for MTA
+image::mta-assessment-new-app-01.png[MTA application inventory]
+
+[start=3]
+. In the form, enter the following information:
+.. *Name* of the new application (mandatory)
+.. *Description* of the application (optional)
+.. *Business service* is thepurpose of the application (optional)
+.. *Tags* software tags that charecterize the application (mandatory)
+.. *Owner* select regitered software owner from drop-down list (optional)
+.. *Contributors* select contributors from drop-down list (optional)
+.. *Comments* enter comments on the migrated application
+. Click *Source code* and enter the following fields.
+.. *Repository type* select *Git* or *Subversion* (mandatory)
+.. *Source Repository* paste the URL of the repository where the software code is saved (mandatory)
+.. *Branch* enter the repository branch that is to be migrated (mandatory)
+.. *Root path* [get info] (optional/mandatory)
+. Click *Binary* and enter the following fields.
+.. *Group* [get info] (optional/mandatory)
+.. *Artifact* [get info] (optional/mandatory)
+.. *Version* software version of the application
+.. *Packaging* [get info] (optional/mandatory)
+. Click *Create*.
+
+.To open a saved application
+
+// Get updated image for MTA
+image::mta-assessment-apps-01.png[MTA aplication inventory]
+
+. Open the {ProductName} user interface and click *Application Inventory* in the left menu bar. *Application assessment* appears in the main pane, listing all the available questionnaires. 
+. Review all the applications and click *Take* to the right of the selected questionnaire. The main pane shows a list of defined assessments.
```

Comment:
How do you get to *Take*? Since these are applications, not questionnaires, I am lost. Is *Take* an option you see by clicking the *Options* menu (kebab.png) ?